addons/binding/org.openhab.binding.silvercrestwifisocket/src/main/java/org/openhab/binding/silvercrestwifisocket/internal/utils/WifiSocketPacketConverter.java

```diff
@@ -111,8 +111,8 @@ public WifiSocketPacketConverter() {
             bEncrypted = this.silvercrestEncryptCipher.doFinal(inputByte);
             int encryptDataLength = bEncrypted.length;
 
-            logger.trace("Encrypted data={" + byteArrayToHexString(inputByte) + "}");
-            logger.trace("Decrypted data={" + byteArrayToHexString(bEncrypted) + "}");
+            logger.trace("Encrypted data={{}}", byteArrayToHexString(inputByte));
+            logger.trace("Decrypted data={{}}", byteArrayToHexString(bEncrypted));
```

Comment:
> SLF4J only cares about the formatting anchor, that is the '{' character immediately followed by '}'. Thus, in case your message contains the '{' or the '}' character, you do not have to do anything special unless the '}' character immediately follows '}'. For example, 

See https://www.slf4j.org/faq.html#logging_performance